seed/index.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+const _ = require('underscore');
```

Comment:
should we align as a team on which libraries we should use? does it matter if teammates use a certain library and others dont? Only thing would be consistent between the different repos but I don't think its that important?

Reply:
@GalexyN as long as we have our dependencies defined in package.json, we will be fine. we'll have to run npm install whenever we pull down each others services.

let me know if you were thinking something else.